### ISSUE TITLE
Missing NuGet.Protocol, version 5.0.2

### ISSUE THREAD

**Issue by sorashi** [cake-contrib/Cake.ExtendedNuGet#34]
```
#addin "nuget:?package=NuGet.Core&version=2.14.0"
#addin "Cake.ExtendedNuGet&version=2.1.1"
```
^
I have these two addins in my build script. When I run the script I get this error:
```
Preparing to run build script...
Running build script...
Could not load .\tools\Addins\Cake.ExtendedNuGet.2.1.1\lib\netstandard2.0\Cake.ExtendedNuGet.dll (missing NuGet.Protocol, Version=5.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35)
./build.cake(65,28): error CS0103: The name 'GetPackageReferences' does not exist in the current context
Error: Error(s) occurred when compiling build script:
./build.cake(65,28): error CS0103: The name 'GetPackageReferences' does not exist in the current context
```

When I add
```
#addin "nuget:?package=NuGet.Protocol&version=5.0.2"
```
to force the package to be downloaded, I get the following output
```
Could not load .\tools\Addins\NuGet.Protocol.5.0.2\lib\netstandard2.0\NuGet.Protocol.dll (missing NuGet.Common, Version=5.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35)
Could not load .\tools\Addins\NuGet.Protocol.5.0.2\lib\netstandard2.0\NuGet.Protocol.dll (missing NuGet.Configuration, Version=5.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35)
Could not load .\tools\Addins\NuGet.Protocol.5.0.2\lib\netstandard2.0\NuGet.Protocol.dll (missing NuGet.Packaging, Version=5.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35)
Could not load .\tools\Addins\NuGet.Protocol.5.0.2\lib\netstandard2.0\NuGet.Protocol.dll (missing NuGet.Versioning, Version=5.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35)
Could not load .\tools\Addins\NuGet.Protocol.5.0.2\lib\netstandard2.0\NuGet.Protocol.dll (missing NuGet.Frameworks, Version=5.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35)
Could not load .\tools\Addins\Cake.ExtendedNuGet.2.1.1\lib\netstandard2.0\Cake.ExtendedNuGet.dll (missing NuGet.Versioning, Version=5.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35))
./build.cake(66,28): error CS0103: The name 'GetPackageReferences' does not exist in the current context
Error: Error(s) occurred when compiling build script:
./build.cake(66,28): error CS0103: The name 'GetPackageReferences' does not exist in the current context
```

Why are these packages missing?

**Comment by TrymBeast** [comment]
I had the same error:
> Could not load .\tools\Addins\Cake.ExtendedNuGet.2.1.1\lib\netstandard2.0\Cake.ExtendedNuGet.dll (missing NuGet.Protocol, Version=5.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35)

But solved it with:
> #addin "NuGet.Protocol&version=5.0.2&loaddependencies=true"

**Comment by sorashi** [comment]
@TrymBeast thanks so much for taking the time to tell me, this works for me too

**Comment by TrymBeast** [comment]
@sorashi No problem! But I think this issue should remain open, Cake.ExtendedNuGet should declare that it depends on NuGet.Protocol package so we don't need to manually declare this dependency in our scripts.

**Comment by sorashi** [comment]
Yes, that's right

**Comment by augustoproiete** [comment]
This was resolved by #37 which bumps NuGet.Protocol to 5.4.0 and will be available in the next release
